l10n/sleha/de-de/xml/article_installation.xml

```diff
@@ -0,0 +1,748 @@
+       <itemizedlist>
+        <listitem>
+         <para>
+          Netzwerkgerätekopplung (bevorzugt)
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          Über einen zweiten Kommunikationskanal in Corosync
+         </para>
+        </listitem>
+       </itemizedlist> </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Knoten-Fencing/STONITH</term>
+     <listitem>  <para>
+      Um das Szenario einer <quote>Systemspaltung</quote> zu verhindern, benötigen Cluster einen Fencing-Mechanismus für Knoten. Wenn eine Systemspaltung vorliegt, werden Cluster-Knoten in zwei oder mehr Gruppen geteilt, die (aufgrund eines Hardware- bzw. Softwarefehlers oder einer unterbrochenen Netzwerkverbindung) keine Kenntnis voneinander haben. Ein Fencing-Mechanismus isoliert den fraglichen Knoten (in der Regel durch das Zurücksetzen oder Ausschalten des Knotens). Dies wird auch als STONITH (<quote>Shoot the other node in the head</quote>, Englisch für „Schieß dem anderen Knoten in den Kopf“) bezeichnet. Als Fencing-Mechanismus für Knoten kann entweder ein physisches Gerät (ein Netzschalter) oder ein Mechanismus wie SBD (STONITH nach Festplatte) in Kombination mit einem Watchdog verwendet werden. Für SBD ist ein gemeinsam genutzter Speicher erforderlich.
+     </para> </listitem>
+    </varlistentry>
+   </variablelist>
+ </sect2>
+  <sect2 xml:id="il-ha-inst-quick-req-sw">
+   <title>Software-Anforderungen</title>
+   <para>
+   Alle Knoten, die Teil des Clusters sind, müssen mindestens über folgende Module und Erweiterungen verfügen:
+  </para>
+
+<itemizedlist>
+   <listitem>
+    <para>Basesystem Module <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></para>
+   </listitem>
+   <listitem>
+    <para>Server Applications Module <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></para>
+   </listitem>
+   <listitem>
+    <para><phrase role="productname"><phrase os="sles">SUSE Linux Enterprise High Availability Extension</phrase></phrase> <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></para>
+   </listitem>
+  </itemizedlist>
+
+  </sect2>
+  <sect2 xml:id="vl-ha-inst-quick-req-other">
+   <title>Sonstige Anforderungen und Empfehlungen</title>
+   <variablelist>
+    <varlistentry>
+     <term>Zeitsynchronisierung</term>  <listitem>
+   <para>
+     Cluster-Knoten müssen mit einem NTP-Server außerhalb des Clusters synchronisiert werden. Ab <phrase role="productname"><phrase os="sles">SUSE Linux Enterprise High Availability Extension</phrase></phrase> 15 ist chrony Standardimplementation von NTP. Sie finden weitere Informationen hierzu im<link xlink:href="https://documentation.suse.com/sles-15/html/SLES-all/cha-ntp.html"><citetitle>Administrationshandbuch</citetitle> für SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">15 SP4</phrase></phrase></link>.
```

Comment:
missing space here: 'hierzu im<link...'

Reply:
Fix now in XML on my local cloned repo and pushed to 'Vistatec/localisation checkin sleha15sp4' branch in repo.

Reply:
![image](https://user-images.githubusercontent.com/68334239/182835724-f4609e62-0992-4c84-840e-057205aeadd3.png)

Fixed now in the TM

![image](https://user-images.githubusercontent.com/68334239/182835872-def01c66-ec6a-4afd-92c8-45a1b8225404.png)